devtools/platform-descriptor-json/src/main/resources/codestarts/amazon-lambda-example/java/src/test/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+quarkus.lambda.handler=test
+
+quarkus.lambda.enable-polling-jvm-mode=true
+
+
```

Comment:
You need to convert this file to `yml`, it is going to be auto-converted at generation to either `yml` or `properties` weither `yaml-config` extension has been selected or not..